projects/nginx/Dockerfile

```diff
@@ -16,12 +16,53 @@
 ################################################################################
 FROM sydr/ubuntu20.04-sydr-fuzz
 
-# Install build dependencies.
-RUN apt-get update && apt-get install -y libpcre3-dev zlib1g-dev mercurial ninja-build liblzma-dev \
-    libz-dev binutils libtool build-essential libssl-dev
+# UPDATE + dep for nginx, preeny
+RUN : \
+    && apt-get update \
+    && apt-get dist-upgrade -y \
+    && apt-get install -y \
+    libpcre3-dev zlib1g-dev mercurial ninja-build liblzma-dev \
+    libz-dev binutils libtool build-essential libssl-dev libseccomp-dev \
+    libini-config-dev \
+    && apt-get autoclean \
+    && :
 
-# Clone target from GitHub. Checkout specified commit. It could be updated later.
-RUN hg clone http://hg.nginx.org/nginx/ -u d1b8568f3042
+# ansifilter
+ARG GIT_ANSIFILTER_TAG="2.18"
+ARG ANSIFILTER_SRC_DIR="/ansifilter"
+RUN : \
+    && git clone --depth=1 https://gitlab.com/saalen/ansifilter.git \
+        -b ${GIT_ANSIFILTER_TAG} ${ANSIFILTER_SRC_DIR} \
+    && cd ${ANSIFILTER_SRC_DIR} \
+    && env CC=clang CXX=clang++ make \
+    && make install \
```

Comment:
Maybe it is better to move installation step to build.sh?

Reply:
Done